rn-cli.config.js

```diff
@@ -8,10 +8,22 @@ const blacklistElements = blacklist( [
 	new RegExp( path.basename( __dirname ) + '/gutenberg/gutenberg-mobile/.*' ),
 	new RegExp( path.basename( __dirname ) + '/react-native-aztec-old-submodule/.*' ),
 ] );
+
```

Comment:
I experimented a bit with this, and it seems the object spread syntax is supported, and the extra modules work with relative paths :tada: :) , so we can do:
```js
module.exports = {
  watchFolders: ['../gutenberg-plugin-hello-world'],
  extraNodeModules: {
    ...enm,
    '@wordpress-mobile/gutenberg-plugin-hello-world': '../gutenberg-plugin-hello-world'
  },
}
```